project/sites/semcommittee/themes/semcommittee_theme/src/scss/components.scss

```diff
@@ -7,9 +7,30 @@
 
 .top_left {
   padding-left: 19rem;
+  @media screen and (max-width: $tab) {
+    padding-left: 0;
+
+    #logo {
+      position: relative;
+    }
+  }
+
 
   #logo img {
     height: 100px;
+    @media screen and (max-width: $tab) {
+      position: relative;
+    }
+  }
+
+  #site-title #site-description {
+    @include adjust-font-size-to(20px);
+    color: #008152;
+    font-style: normal;
```

Comment:
If you adding a new font you need to reference it 1st in your .info/.libraries files https://github.com/dof-dss/nicsdru_unity_theme/blob/development/nicsdru_unity_theme.info.yml#L9 and https://github.com/dof-dss/nicsdru_unity_theme/blob/development/nicsdru_unity_theme.libraries.yml#L43. Then you can add it to your subtheme e.g. https://github.com/dof-dss/nicsdru_unity_theme/blob/development/src/scss/init/_typography.scss#L22